app/src/main/java/com/sunasterisk/food_01/screen/main/tabLayout/TabFragment.kt

```diff
@@ -0,0 +1,63 @@
+package com.sunasterisk.food_01.screen.main.tabLayout
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.sunasterisk.food_01.R
+import com.sunasterisk.food_01.screen.adapter.ViewPagerAdapter
+import com.sunasterisk.food_01.screen.main.tabLayout.detailFragment.DetailFragment
+import com.sunasterisk.food_01.screen.main.tabLayout.favoriteFragment.FavoriteFragment
+import com.sunasterisk.food_01.screen.main.tabLayout.homeFragment.HomeFragment
+import com.sunasterisk.food_01.screen.main.tabLayout.searchFragment.SearchFragment
+import kotlinx.android.synthetic.main.fragment_tab.*
+
+class TabFragment : Fragment() {
+    val homeFragment = HomeFragment()
+    val favoriteFragment = FavoriteFragment()
+    val searchFragment = SearchFragment()
+    val detailFragment = DetailFragment()
+    override fun onCreateView(
```

Comment:
```suggestion

    override fun onCreateView(
```
function thì cách ở trên một dòng nha